CMakeLists.txt

```diff
@@ -1,40 +1,62 @@
 cmake_minimum_required(VERSION 3.0)
 project(FugDemos)
 
+
+#Target to build
+set(FUGDEMOS_TARGET "Breakout" CACHE STRING "Demo project to build")
+
+
+#Set required C++ standard
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+#Output libraries to same directory with binaries
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
-add_subdirectory(ext)
-add_subdirectory(breakout)
```

Comment:
Name change of the breakout directory should be in the previous commit, which is currently broken.

Reply:
Also CMake copies the source tree directory structure into the build folder and on Unix-like systems executables have no file extension. These two factors create a situtaion where we have build/Breakout directory while linker tries (and fails) to create an executable with the exact same name.